test/e2e/certificate.spec.ts

```diff
@@ -0,0 +1,38 @@
+import { test, expect } from '@playwright/test';
+import { loggedInAsUser } from './utils';
+
+test.describe('Certificate Page', () => {
+  loggedInAsUser();
+
+  test.beforeEach(async ({ page }) => {
+    await page.goto('/certificate');
+  });
+
+  test('Request a Certificate', async ({ page }) => {
+    const titleHeader = page.getByRole('heading', { name: 'Certificate' });
+    const domainName = page.getByText('user1.starchart.com');
+
+    await expect(domainName).toContainText('user1.starchart.com');
+    await expect(titleHeader).toContainText('Certificate');
+    await page.getByRole('button', { name: 'Request a Certificate' }).click();
+
+    page
+      .locator('div')
+      .filter({
+        hasText: 'We have received your request, and will notify you when your certificate is read',
+      })
+      .nth(1);
```

Comment:
Could you please explain what this does? From what I understand, this simply gets the div, and doesn't do anything with it?

Reply:
oh that's actually a good catch. I was planning on finding the div to check if the request has rendered the loading page. I didn't do an expect to check that fact so it looks weird having it there